src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/fakta/BehandlingFaktaDto.kt

```diff
@@ -15,8 +15,8 @@ import java.time.LocalDateTime
 data class BehandlingFaktaDto(
     val søknadMottattTidspunkt: LocalDateTime?,
     val hovedytelse: FaktaHovedytelse,
-    val aktivitet: FaktaAktivtet,
-    val barn: List<FaktaBarn>,
+    val aktivitet: FaktaAktivtet?,
```

Comment:
Burde vi vurdere 2 ulike DTO'er for tilsynbarn og læremidler?

Reply:
Hva syns du om løsningen slik det er implementert nå? hentFakta-endepunktet returnerer "riktig" type DTO, men vi er fortsatt nødt til å type-sjekke i frontend.

Reply:
Denne PR kan sees i sammenheng med denne: https://github.com/navikt/tilleggsstonader-sak-frontend/pull/523

Reply:
Tenker man ikke trenger 2 ulike endepunkter, typescript håndterer union type